### DISCUSSION TITLE
Reading information from socket failed (Apple Homepod mini as Thread Border Router)

### DISCUSSION THREAD

**Discussion by maticcavalleri** [openthread/openthread#9852] [Q&A] [answered]
I'm trying to start a Thread network. I have a Apple Homepod mini on Thread 1.3 which should be cross compatible between the different vendors.

I see ot-commissioner says "Implements both the Thread 1.1 and 1.2 Commissioner", Is this the reason why I'm unable to start the thread network with the Homepod mini which is on Thread 1.3?

I made sure all of the steps to fix error "Reading information from the socket failed" were tried (using correct port, network...) but I keep getting this error.

To replicate:
I have raspberry pi and Homepod mini connected to same wifi. With RPI I execute the following commands:
1. commissioner-cli (start the tool)
2. config set pskc abc123
3. br scan --timeout 2000 --export border-routers.json
4. br ad border-routers.json
5. network select MyHome727597419
6. start

On start command i get:
MyHome727597419> start
E2AB16A12406451D:
no updates to DTLS configuration, default configuration will be used

IO_ERROR: NET - Reading information from the socket failed
[failed]

How can I use Homepod mini as a Thread border router?

**Comment by jwhui** [comment, marked as answer]
Gaining access to a Thread Border Router requires retrieving the Thread credentials. AFAIK, the only way to retrieve such credentials from an Apple Thread Border Router is by using the [iOS ThreadNetwork APIs](https://developer.apple.com/documentation/threadnetwork/).
